application/domain/domain.go

```diff
@@ -515,3 +513,11 @@ func checkSamlConfig(env *EnvStruct) {
 		panic("required SAML variable SamlSpPrivateKey is undefined")
 	}
 }
+
+// TODO: replace this with the builtin function after upgrading to Go 1.21
+func Max(a, b int) int {
```

Comment:
I don't think this is worth it, but mentioning it anyways.

Alternately, could use a generic with `"golang.org/x/exp/constraints"` package or by defining own input (suggest something along the lines of `int|api.Currency`).

```suggestion
func Max[T constraints.Ordered](a, b T) T {
```

Reply:
It's a worthy mention, but I hope we won't be waiting long until Buffalo is ready for Go 1.21